tests/Integration/Frameworks/Symfony/Version_3_4/.gitignore

```diff
@@ -0,0 +1,17 @@
+/.web-server-pid
```

Comment:
Beyond this point is just skeleton apps as added by `composer create-project` script.